app/views/logix/about.html.erb

```diff
@@ -21,7 +21,7 @@
                       </p>
                       <p>
                         Join our chat room @
-                        <a href="https://gitter.im/CircuitVerse/community" target="_blank">Gitter Channel</a>
+                        <a href="https://join.slack.com/t/circuitverse-team/shared_invite/enQtNjc4MzcyNDE5OTA3LTdjYTM5NjFiZWZlZGI2MmU1MmYzYzczNmZlZDg5MjYxYmQ4ODRjMjQxM2UyMWI5ODUzODQzMDU2ZDEzNjI4NmE" target="_blank">Slack</a>
```

Comment:
Can we store this in a constant instead of hardcoding in the view?

Reply:
Done :rocket: